lib/assets/javascripts/views/questions/show.js

```diff
@@ -6,18 +6,15 @@ $(() => {
     $(target).show();
   });
 
-  $('.add_annotations_button').on('click', (e) => {
-    const source = e.target;
-    const target = $(source).attr('href');
-    $(target).show();
-    $(source).hide();
+  $('.annotations_button').on('click', (e) => {
+    e.preventDefault();
+    $($(e.target).attr('href')).show();
```

Comment:
please can you extract $(e.target) as a constant (e.g. const target = $(e.target) otherwise we read the DOM element twice.